README.md

```diff
@@ -66,27 +66,27 @@ in flake-utils-plus.lib.systemFlake {
 
 
   ################
-  ### channels ###
+  ### streams ###
   ################
```

Comment:
Misaligned lines 68 and 70 :D

Reply:
> What is the reason for choosing word stream? Is this official?

Just some brainstorming. Also considered `nubs`, `pkgsets`, `stubs`, `sources`and `pkgstreams`. Decided since streams we're related to channels the association should be obvious enough.

As far as being official, well there is:
NixOS/nix#e2af11ce071fc109ad3e517bf15a87024d0a9ae5

Which explicitly removed channels from the manpages in favour of pulling from nixpkgs directly. I could have swore that, when flakes were brand new, there was some discussion about the move away from channels, but I can't seem to find it anywhere so it might be my imagination. It was about 2 years ago after all.